src/__tests__/Search/CompareOverTime.test.tsx

```diff
@@ -194,4 +185,63 @@ describe('Compare Over Time', () => {
     await user.click(labelTime[0]);
     expect(comment[0]).not.toBeInTheDocument();
   });
+
+  it('should remove the checked revision once X button is clicked', async () => {
+    renderSearchViewComponent();
+    await expandOverTimeComponent();
+
+    // set delay to null to prevent test time-out due to useFakeTimers
+    const user = userEvent.setup({ delay: null });
+
+    const searchInput = screen.getAllByRole('textbox')[2];
+    await user.click(searchInput);
+    const checkbox = (await screen.findAllByTestId('checkbox-0'))[0];
+    await user.click(checkbox);
+    expect(checkbox).toHaveClass('Mui-checked');
+    const removeButton = document.querySelectorAll('[title="remove revision"]');
+    expect(removeButton[0]).toBeInTheDocument();
+    expect(screen.getAllByTestId('selected-rev-item')[0]).toBeInTheDocument();
+
+    await user.click(removeButton[0]);
+
+    expect(document.body).toMatchSnapshot();
```

Comment:
optional: Maybe it would be good to run a snapshot for just the compare over time component. This would give smaller snapshots, probably faster tests, and easier to review snapshots.